packages/ui/components/DataFeedDetails.vue

```diff
@@ -13,14 +13,18 @@
       @change-range="updateQuery"
     />
     <DataFeedDescription
+      :is-routed="normalizedFeed.isRouted"
       :feed-name="normalizedFeed.name"
       :network-name="normalizedFeed.networkName"
       :last-result-value="lastResultValue"
       :last-result-date="lastResultDate"
       :feed-time-to-update="feedTimeToUpdate"
       :deviation="normalizedFeed.deviation"
     />
-    <Fieldset :title="$t('data_feed_details.trigger_parameters')">
+    <Fieldset
+      v-if="maxTimeToResolve || normalizedFeed.deviation"
```

Comment:
Is this equivalent to check if it's routed?
```suggestion
      v-if="!normalizedFeed.isRouted"
```

Reply:
it is not, it could be a routed data feed with a heartbeat